src/webgl/3d_primitives.js

```diff
@@ -1633,6 +1724,7 @@ p5.RendererGL.prototype.bezierVertex = function(...args) {
           w_z[3] * this._lookUpTableBezier[i][3];
         this.vertex(_x, _y, _z);
       }
+      this.curFillColor = finalFillColor; // Set curFillColor to finalFillColor
```

Comment:
Maybe instead of the comment describing what this line does, we could instead describe why it needs to do that? e.g. so that we leave `curFillColor` with the last value the user set it to?

Reply:
My comment here was too easy...I'll fix it!